.github/workflows/scan.yaml

```diff
@@ -0,0 +1,144 @@
+name: Trivy Scan
+on:
+  workflow_dispatch: {}
+  schedule:
+    # run every day at 3:07am UTC
+    - cron: '7 3 * * *'
+
+env:
+  GOMPLATE_VERSION: v3.11.4
+  DOCKER_USR: ${{ secrets.DOCKER_USR }}
+
+jobs:
+  generate-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      versions: ${{ steps.get-releases.outputs.versions}}
+      supported_releases: ${{ steps.get-releases.outputs.supported_releases }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3
+        with:
+          fetch-depth: 0
+
+      - name: Get last 3 supported releases and their latest tags
```

Comment:
Nit: I think all of our other steps use brief title case names.

Reply:
Changed all of them to be shorter and title case 👍